packages/fastapi/src/executors/serve/executor.ts

```diff
@@ -22,6 +27,16 @@ export default async function* runExecutor(
       }
       return CmdStatus.Continue;
     };
+    const dbMigrate = executePoetryCommand(
+      projectRoot + '/src',
```

Comment:
Should use `joinPathFragments` helper for platform compat, e.g. `joinPathFragments(projectRoot, src)`